src/DIRAC/Core/Security/Locations.py

```diff
@@ -12,6 +12,23 @@
 g_SecurityConfPath = "/DIRAC/Security"
 
 
+def getTokenLocation():
+  """ Get the path of the currently active access token file
+  """
+  envVar = 'DIRAC_TOKEN_FILE'
+  if envVar in os.environ:
+    tokenPath = os.path.realpath(os.environ[envVar])
+    if os.path.isfile(tokenPath):
+      return tokenPath
+  # /tmp/JWTup_u<uid>
+  tokenName = "JWTup_u%d" % os.getuid()
+  if os.path.isfile("/tmp/%s" % tokenName):
+    return "/tmp/%s" % tokenName
+
+  # No access token found
+  return False
```

Comment:
I think this should always return a string to avoid needing `getTokenLocation() or ....`

Reply:
I do not really understand in which case it will help, we still need to understand whether the file is found or not, no?

Reply:
But it keeps the logic for declaring where the file is in one place. At the moment it's duplicated in at least three places.